packages/flutter_tools/test/src/common.dart

```diff
@@ -137,3 +139,55 @@ Future<void> expectToolExitLater(Future<dynamic> future, Matcher messageMatcher)
     fail('ToolExit expected, got $e\n$trace');
   }
 }
+
+@isTest
+void test(String description, FutureOr<void> body(), {
```

Comment:
Will this wrapper be in place for all tool tests that just say `test()` ?

Reply:
Yes, we could take this opportunity to enforce some reasonable default behavior in the future if necessary, such as providing zone based throwing implementations of the real filesystem or network behavior.